rpc/rpc-types/src/lib.rs

```diff
@@ -1 +0,0 @@
-
```

Comment:
The diff is `rpc/rpc-types/src/lib.rs` (empty file) -> `rpc/types/src/lib.rs` (non-empty file).

It doesn't show it as a move since stuff has been added I guess.